appveyor.yml

```diff
@@ -0,0 +1,17 @@
+version: 10.2-{build}
+before_build:
+- md %APPVEYOR_BUILD_FOLDER%\win_build
+- cd %APPVEYOR_BUILD_FOLDER%\win_build
+- cmake ..  -G "Visual Studio 15 2017" -DBOOST_ROOT=C:\Libraries\boost_1_64_0 -DPKG_CONFIG_EXECUTABLE=C:\perl\bin\pkg-config -DBISON_EXECUTABLE=C:\cygwin\bin\bison
```

Comment:
PKG_CONFIG does not really work on Windows, Strawberry perl's uses mingw  libraries, which VS  compiler cannot use,
so PKG_CONFIG_EXECUTABLE is probably best NOT-FOUND

Reply:
Also, BOOST_ROOT is not really needed. Why it is there?

Reply:
Thanks @vaintroub. As usual your expertise is invaluable.